docs/decisions/infra.md

```
@@ -69,13 +69,14 @@ Cloudflare를 리버스 프록시로 사용한다. 선택한 이유는 크게 
 Docker 안에 넣지 않은 이유는 아래와 같다.
 
 - Frontend는 Vite 빌드 결과물(`dist/`)을 Nginx가 직접 서빙 - Docker 볼륨 마운트 없이 단순
-- 설정 파일 3개로 역할이 명확히 분리됨
+- 설정 파일 4개로 역할이 명확히 분리됨
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)